frontend/kesaseteli/employer/src/components/footer/Footer.sc.ts

```diff
@@ -0,0 +1,18 @@
+import { Footer as HdsFooter } from 'hds-react';
```

Comment:
copy-pasted from benefit's code. Perhaps this could be moved to shared component folder?